classes/Mail.php

```diff
@@ -465,7 +465,7 @@ function ($carry, $item) {
             );
 
             /* Create mail and attach differents parts */
-            $subject = '[' . Tools::safeOutput($configuration['PS_SHOP_NAME']) . '] ' . $subject;
+            $subject = '[' . strip_tags($configuration['PS_SHOP_NAME']) . '] ' . $subject;
```

Comment:
Why do you need to strip tags in an email subject ? 

(btw, yo @Progi1984  🕺 )

Reply:
It's a security. Someone could put an html tag in shop name.